content/en/docs/concepts/scheduling-eviction/scheduling-framework.md

```diff
@@ -43,15 +43,18 @@ A scheduling or binding cycle can be aborted if the Pod is determined to
 be unschedulable or if there is an internal error. The Pod will be returned to
 the queue and retried.
 
-## Extension points
+## Interfaces
 
-The following picture shows the scheduling context of a Pod and the extension
-points that the scheduling framework exposes. In this picture "Filter" is
+The following picture shows the scheduling context of a Pod and the interfaces
+that the scheduling framework exposes. In this picture "Filter" is
 equivalent to "Predicate" and "Scoring" is equivalent to "Priority function".
```

Comment:
```suggestion
that the scheduling framework exposes.
```

It's been a while since we last used the terms "predicate" and "priority". We can get rid of this sentence to avoid confusion.